siac/rentercmd.go

```diff
@@ -274,7 +275,15 @@ func rentersetallowancecmd(amount, period string) {
 	if err != nil {
 		die("Could not parse period")
 	}
-	err = post("/renter", fmt.Sprintf("funds=%s&period=%s", hastings, blocks))
+	_, err = strconv.Atoi(hosts)
+	if err != nil {
+		die("Count not parse host count")
+	}
+	_, err = parsePeriod(renewWindow)
```

Comment:
can you help me understand why the return value of parsePeriod is ignored here but used with blocks above @DavidVorick  ?  Seems like two different conventions going into the Sprintf but i don't understand the benefit.